.github/PULL_REQUEST_TEMPLATE/opt_sprint.md

```
@@ -42,6 +42,69 @@ Explicitly list what is intentionally NOT being done in this PR.
 
 - Issues/PRs to do next:
 
+## Column Graph Native Workstream (#1646, if applicable)
+
+### Copied/Adapted From Old Stack
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)